powerpacks.go

```diff
@@ -300,6 +372,18 @@ func (service *PowerpackService) Find_shortcode(shortcode string) (response *Fin
 	return
 }
 
+func (service *PowerpackService) Find_tollfree(tollfree string) (response *FindTollfreeResponse, err error) {
```

Comment:
This function (and all other exported functions in this file)  should not be using `_` but be called e.g.`FindTollfree`.